frontend/ui/widget/inputtext.lua

```diff
@@ -584,6 +586,12 @@ function InputText:focus()
     self._frame_textwidget.color = Blitbuffer.COLOR_BLACK
 end
 
+local sym_key_map = {
```

Comment:
```suggestion
-- NOTE: This key_map is to be used by kindles with keyboards, it is 'inspired' by the symbol layer on the virtual keyboard but,
--       we have taken the liberty of making some adjustments since:
--       * K3 does not have a numeric row and, 
--       * we want to prioritise the most likely-used characters for "style tweaks" and text-writing 
--       (in English, sorry everybody else, there are just not enough keys)
local sym_key_map = {
```